cardano-node/README.md

```diff
@@ -100,9 +203,11 @@ Usage: cardano-cli --real-pbft migrate-delegate-key-from (--byron-legacy | --rea
   Migrate a delegate key from an older system version.
 
 Available options:
-  --byron-legacy           Use the Byron/Ouroboros Classic suite of algorithms
-  --real-pbft              Use the Permissive BFT consensus algorithm using the
-                           real ledger
+  --byron-legacy           Byron/Ouroboros Classic suite of algorithms
```

Comment:
Which one does this correspond to?

Reply:
Resolved in slack: https://github.com/input-output-hk/cardano-node/issues/371